src/de/grobox/liberario/TransportNetwork.java

```diff
@@ -57,6 +57,12 @@ public TransportNetwork setDescription(@Nullable String description) {
 		return this;
 	}
 
+	public TransportNetwork setDescription(@Nullable String description, @Nullable String networks) {
```

Comment:
It will use the English string. So you could remove the annotation.

Btw. you can also squash commits yourself and force push into your branch. But when merging they will all be squashed into one anyway now.